app/src/main/java/com/animal_villa/dto/Animal.kt

```diff
@@ -1,4 +1,7 @@
 package com.animal_villa.dto
 
+import androidx.room.Entity
+
+@Entity
 class Animal {
 }
```

Comment:
This isn't a data class, and there aren't any fields here to persist, so @Entity does not make sense in this context.